go/libkb/kbfs_folder.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"strings"
+)
+
+var caseSensitiveServices = map[string]bool{"hackernews": true}
```

Comment:
I've made a habit of putting all HackerNews-related code into `proof_support_hackernews`.  This way when we add another service, we don't have stuff peppered all over the repo...